x/nft/keeper/keys.go

```diff
@@ -48,16 +49,43 @@ func classTotalSupply(classID string) []byte {
 
 // nftOfClassByOwnerStoreKey returns the byte representation of the nft owner
 // Items are stored with the following key: values
-// 0x03<owner><classID><Delimiter(1 Byte)>
+// 0x03<owner><Delimiter(1 Byte)><classID><Delimiter(1 Byte)>
 func nftOfClassByOwnerStoreKey(owner sdk.AccAddress, classID string) []byte {
 	owner = address.MustLengthPrefix(owner)
 	classIDBz := conv.UnsafeStrToBytes(classID)
 
-	var key = make([]byte, len(NFTOfClassByOwnerKey)+len(owner)+len(classIDBz)+len(Delimiter))
+	var key = make([]byte, len(NFTOfClassByOwnerKey)+len(owner)+len(Delimiter)+len(classIDBz)+len(Delimiter))
 	copy(key, NFTOfClassByOwnerKey)
 	copy(key[len(NFTOfClassByOwnerKey):], owner)
-	copy(key[len(NFTOfClassByOwnerKey)+len(owner):], classIDBz)
-	return append(key, Delimiter...)
+	copy(key[len(NFTOfClassByOwnerKey)+len(owner):], Delimiter)
+	copy(key[len(NFTOfClassByOwnerKey)+len(owner)+len(Delimiter):], classIDBz)
+	copy(key[len(NFTOfClassByOwnerKey)+len(owner)+len(Delimiter)+len(classIDBz):], Delimiter)
+	return key
+}
+
+// prefixNftOfClassByOwnerStoreKey returns the prefix of the result of the method nftOfClassByOwnerStoreKey
+// Items are stored with the following key: values
+// 0x03<owner><Delimiter>
+func prefixNftOfClassByOwnerStoreKey(owner sdk.AccAddress) []byte {
+	owner = address.MustLengthPrefix(owner)
+
+	var key = make([]byte, len(NFTOfClassByOwnerKey)+len(owner)+len(Delimiter))
+	copy(key, NFTOfClassByOwnerKey)
+	copy(key[len(NFTOfClassByOwnerKey):], owner)
+	copy(key[len(NFTOfClassByOwnerKey)+len(owner):], Delimiter)
+	return key
+}
+
+// Note: the full path of the nftOfClassByOwnerStoreKey stored in the store: 0x03<owner><Delimiter><classID><Delimiter><nftID>,
+// the key of the prefix store query result constructed using the prefixNftOfClassByOwnerStoreKey function needs to remove the 0x03<owner><Delimiter> prefix
+func parseNftOfClassByOwnerStoreKey(key []byte) (classID,nftID string) {
```

Comment:
make sure the gofmt is run on the files.
```suggestion
func parseNftOfClassByOwnerStoreKey(key []byte) (classID, nftID string) {
```